conductr_cli/bndl_create.py

```diff
@@ -35,14 +36,17 @@ def bndl_create(args):
 
         return 2
 
-    buff_in = BufferedReader(sys.stdin.buffer, IO_CHUNK_SIZE) if args.source is None else None
+    if args.source is None and not sys.stdin.isatty:
+        buff_in = BufferedReader(sys.stdin.buffer, IO_CHUNK_SIZE)
+    else:
+        buff_in = None
```

Comment:
This change is necessary given that the `bndl` command now does not require an input, e.g. when creating a new bundle configuration from scratch.